reports/TC39/2017-01.md

```diff
@@ -0,0 +1,124 @@
+
+Without the null propagation operator:
+
+```js
+const firstName = (message
+  && message.body
+  && message.body.user
+  && message.body.user.firstName) || 'default';
+```
+
+With the null propagation operator:
+```js
+const firstName = message.body?.user?.firstName || 'default';
+```
+
+As you can see, the null propagation operator avoids constant checks for falsiness before navigating down an object graph, saving the developer substantial effort.
+As of right now, expect the null propagation operator to return `null` if it reaches a `null` property, and `undefined` if it reaches an `undefined` property.
+
+This proposal advanced to stage 1 with overwhelming support from the committee given it's usefulness. That said, there is substantial concern about the behavior of this
+operator in several edge cases. This is a common complexity when adding syntax to a language.
+
+### RegExp Lookbehind
+
+Regular expression lookbehinds are a common feature in the regex implementations of several programming languages including .NET, Python, PHP, Go, and Ruby.
+[This proposal](https://github.com/tc39/proposal-regexp-lookbehind) creates a regular expression syntax that allows developers to ensure that a specific pattern either
+precedes or DOES NOT precede a capture group, without actually capturing that pattern.
+
+One example scenario where this is useful is when capturing amounts of money. One can use a positive lookbehind to make sure that a dollar sign precedes an amount, by
+using the `(?<=...)` assertion. Completing the regular expression to look for digits, followed by a decimal, and then more digits we arrive at `/(?<=\$)\d+(\.\d*)?/`.
+This expression will match `$12.25` and return `12.15`.
+
+If one wanted to ensure that `$` did not precede the number, one could use a negative lookbehind. This syntax is denoted as `(?<!...)`. Assuming that one wanted to find
+all decimal values that were not US dollars, one could change the previous regular expression to `/(?<!\$)\d+(\.\d*)?/`. This would match `€10.53` but not `$10.53`.
+
+This proposal advanced to stage 2 at this meeting, meaning that initial spec text is available.
+
+### 64 Bit Integers / BigInts
+
+I will not go deeply into [this proposal](https://gist.github.com/BrendanEich/4294d5c212a6d2254703), but I want to note that for OS interoperability purposes,
+the Node.js team has a strong need for 64 bit integers. In addition to these use cases, there are significant usages for 64 bit integers in cryptography and gaming.
+
+There is a feeling on the committee that it may be better to implement a bigint type - that is to say a type that is not limited to 64 bits, and can be optimized
+to the appropriate size based on the code, variables, and underlying system as appropriate. However, this presents more work in specification. If anybody has specific
+arguments about why a bigint type would be strongly preferable to an int64 type, we would be interested to hear your argument.
```

Comment:
It wasn't clear after discussion that it would be more work - BE is going to investigate whether speccing BigInt is feasible in his time budget.